.docs/index.md

```diff
@@ -138,7 +141,7 @@ properly. For instance, Docker communicates to the REX-Ray Volume Driver via
 a UNIX socket file.
 
 ```bash
-$ rexray start
+$ rexray service start
```

Comment:
Hi @kacole2,

Why do you prefer the `service` command? If it's so it falls in line with the visible commands available to REX-Ray's online help, I can get behind that. I'm just curious.